### PR TITLE
PROD - Check Informix for reviews, if we're missing that data

### PR DESCRIPTION
**Pull request by jmgasper**
If there are no reviews from informix, this PR implements loading them from informix into DynamoDB and ES for future use.

https://topcoder.atlassian.net/browse/PS-268

**NOTE** - The Informix driver for Node.js does _not_ support Node >14, so we had to downgrade the version used in the package.json.

### CODE REVIEW

**Comment by testflyjets** [comment]
> **NOTE** - The Informix driver for Node.js does _not_ support Node >14, so we had to downgrade the version used in the package.json.

Ugh. If I could snap my fingers and do one thing in Topcoder, it would be to get off stinkin' Informix.

File: src/common/informixHelper.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+/*
+ * This function loads the online review details for a given submission from Informix. 
+ * It uses the data to create review and reviewSummation objects which are then saved
+ * back to DynamoDB through the relevant services, and to ES through Bus API messages
+ * processed by the submission-processor-es code
+ */
+async function loadOnlineReviewDetails (authUser, submission) {
+  const reviewSummation = {}
+  const reviewsCreated = []
+  // We can only load in OR details from the legacy submission ID.
```
Comment (testflyjets) [inline src/common/informixHelper.js:19]: Can we add some spaces between the major blocks of code here, just to help my old, tired eyes more easily scan it?

File: src/common/informixHelper.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+function queryInformix (query) {
+  const connectionString = 'SERVER=' + config.get('INFORMIX.SERVER') +
+    ';DATABASE=' + config.get('INFORMIX.DATABASE') +
+    ';HOST=' + config.get('INFORMIX.HOST') +
+    ';Protocol=' + config.get('INFORMIX.PROTOCOL') +
+    ';SERVICE=' + config.get('INFORMIX.PORT') +
+    ';DB_LOCALE=' + config.get('INFORMIX.DB_LOCALE') +
+    ';UID=' + config.get('INFORMIX.USER') +
+    ';PWD=' + config.get('INFORMIX.PASSWORD')
+  let result = null
```
Comment (testflyjets) [inline src/common/informixHelper.js:85]: Does a `try/catch` make any sense here to trap DB errors?

File: src/services/SubmissionService.js [hunk 3]
```diff
@@ -247,11 +248,30 @@ async function listSubmissions (authUser, query) {
...
+      _.forEach(submission.review, (review) => {
+        if (review.metadata && review.metadata.source) {
+          if (review.metadata.source === 'Online Review') {
+            hasReviewInES = true
+          }
+        }
+      })
+    }
+
+    // We don't have the Online Review details in ES, so go to Informix to get them and
```
Comment (testflyjets) [inline src/services/SubmissionService.js:264]: Really excellent comment for anyone coming to this months or years from now -- nice work.

**Review by testflyjets — DISMISSED** [review]
Minor formatting suggestions.

File: src/common/informixHelper.js (hunk 2 above)
Comment (jmgasper) [reply src/common/informixHelper.js:85]: Done

**Pull request merged**